libraries/AP_FlashStorage/examples/FlashTest/FlashTest.cpp

```diff
@@ -154,7 +154,10 @@ void FlashTest::setup(void)
     if (memcmp(mem_buffer, mem_mirror, sizeof(mem_buffer)) != 0) {
         AP_HAL::panic("FATAL: data mis-match");
     }
-    AP_HAL::panic("TEST PASSED");
+    while (true) {
+        hal.console->printf("TEST PASSED");
+        hal.scheduler->delay(20000);
+    }
```

Comment:
If we want a loop why not put this in the loop method? That's exactly what it is there for...

Reply:
Moved everything into the loop method.